core/views.py

```diff
@@ -84,6 +84,16 @@ def get_success_url(self: "PostCreateView") -> str:
         return reverse("post-detail", kwargs={"pk": self.object.pk})
 
 
+class PostDeleteView(DeleteView):
+    model = Post
+    template_name = "core/post-confirm-delete.html"
+    success_url = reverse_lazy("post-list")
```

Comment:
Why user is forced to go to post-list? i think it would be better to redirect to the page that was before the button "delete" was pressed